stream_alert_cli/test/results.py

```diff
@@ -135,10 +147,13 @@ def __str__(self):
         )
 
         if self._test_event.get('validate_schema_only'):
-            line = 'Validation Only: True'
-            template += '\n' + line.rjust(len(line) + self._DEFAULT_INDENT * 3)
-        elif self._with_rules:
-            template += self._RULES_TEMPLATE
+            template += self._VALIDATION_ONLY
+
+        if self.rules_run:
```

Comment:
The `elif` is implied by `@property def rules_run`, because that code has a check for `validate_schema_only`.

I removed an `elif` from before because it was making it hard to make the code more modular (because now the `publisher` results depended on the `validate_schema_only` code, etc). At this point the tests have already been run, this is just the code to render the results, so there's minimal performance losses from having this here.